pandunia-lekse.csv

```diff
@@ -2329,69 +2342,60 @@ paxu||||suffer (get hurt)||sufrir||||||||||||||sattua (käydä kipeää)|cierpie
 peda||||fart (flatulate)|péter|peer|peidar|пердеть||放屁|おならをする|방귀뀌다||पादना|পাদা|||||pierrä (pieraista)|pierdzieć
 pedale|||deu:eng:spa:por:tur:pedal, fra:pédale, rus:педаль (pedal'), pol:pedał, jpn:ペダル (pedaru), kor:페달 (pedal)|pedal||pedal|||||||||||||pedalo|poljin (pedaali)|pedał
-pedalosikle||||bicycle (pedal bike)||bicicleta||||||||||||||polkupyörä|rower z pedałami
+pedalsikle||||bicycle (pedal bike)||bicicleta||||||||||||||polkupyörä|rower z pedałami
 pede|||ben:পাদ (pad), hin:पाद (pād), spa:pedo, por:peido, fra:pet, rus:пердение (perdeniye)|fart (flatulence)|pet (vesse)|pedo|peido|пердение||屁|屁|방귀|đánh rắm|पाद||kentut|shuzi|osuruk|furzo|pieru|pierdnięcie
 pelmene|||rus:пельмень (pel'men'), pol:pielmeni, eng:pelmeni|dumpling (ravioli, pelmeni)||empanada (pelmeni)||||饺子||||||||||myky (pelmeni, ravioli)|kluska, knedel, pieróg; ravioli, pelmeni
 pelpele|||kat:პეპელა (ṗeṗela), mao:pēpepe, swa:kipepeo, bul:пеперуда (peperuda), yor:labalábá|butterfly||mariposa|||||||||||kipepeo||papilio|perhonen|motyl
-penda||||hang (dangle)||colgar||||||||||||||roikuttaa|powiesić, wieszać
-penise||||penis (phallus)||pene|||||||||||||peniso|siitin (penis)|penis
+penda|||eng:pendant, por:fra:pender|hang (dangle)||colgar||||||||||||||roikuttaa|powiesić, wieszać
+penise|||eng:tur:may:penis, spa:pene, por:fra:pénis, rus:пенис (penis)|penis (phallus)||pene|||||||||||||peniso|siitin (penis)|penis
 penta|||eng:paint, fra:peintre|paint||pintar|||||||||||||pentri|maalata|namalować, malować
 penter||||painter||pintor||||||||||||||maalari|malarz
 pepine|||spa:por:pepino, tgl:pipino, sin:පිපිඤ්ඤා (pipiññā)|cucumber||pepino|||||||||||||kukumo|kurkku|ogórek
 pere|||hin:पर (par), fas:urd:(par), kur:perr, rus:перо (pero)|feather||pluma||перо||||||पर्दा|||||plumo|sulka|pióro
 peruvia|dex|PE||Peru||Perú||||||||||||||Peru|Peru
 pese|||hin:पैसा (paisā), mar:पैसा (paisā), pnb:ਪੈਸਾ (pēsā), guj:પૈસો (paiso), ara:(bēzāt), swa:pesa, (spa:peso)|money|argent|dinero (plata)|dinheiro|деньги|فلوس‎|钱|金|||पैसा||uang (wang)|pesa||mono|raha|pieniądze
-pesodiske||||coin|monnaie (pièce)|moneda|moeda|монета||硬币||||सिक्का||keping||madeni para|monero|kolikko|moneta
+pesdiske||||coin|monnaie (pièce)|moneda|moeda|монета||硬币||||सिक्का||keping||madeni para|monero|kolikko|moneta
 pete|badan||ben:পেট (peṭ), hin:पेट (peṭ), may:perut, ara:(baṭn)|belly (stomach, abdomen)||barriga (vientre, panza)||||||||पेट (उदर)|পেট|perut|||ventro|maha (vatsa)|brzuch
-petre|||hin:पत्थर (patthar), urd:(patthar), ita:pietra, por:pedra, spa:piedra, fra:pierre|rock|roche|roca|rocha|горная порода|||||||||||roko|kallio|skała
+petre|||hin:पत्थर (patthar), urd:(patthar), ita:pietra, por:pedra, spa:piedra, fra:pierre|rock (stone)|roche (pierre)|roca (piedra)|rocha (pedra)|горная порода||||石 (岩)|石||पत्थर||||roko (ŝtono)|kivi (kallio, murikka)|skała (kamień)
```

Comment:
My idea was to differentiate "petre" and "seke". "petre" would be the mineral matter and "seke" would be a singular piece of it. Can you imagine a difference between "petroxane" (mountain of rock) and "sekoxane" (mountain of stones)?
I also though of additional words from "seke" (stone) like "lilseke" (pebble) and "dayseke" (boulder). These words could be used for other materials besides rock. For example "aurseke" would be a nugget of gold.